### PR TITLE
{bio}[GCC/10.2.0] freebayes v1.3.5, SeqLib v1.2.0, fermi-lite v20190320, SSW v1.1

### PR DESCRIPTION
**Pull request by deniskristak**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/filevercmp/filevercmp-20191210-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+homepage = 'https://github.com/ekg/filevercmp'
+description = """filevercmp function as in sort --version-sort."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+github_account = 'ekg'
+source_urls = [GITHUB_SOURCE]
+sources = ['%s.tar.gz' % local_commit]
+patches = ['%(name)s-20191210_build-libs.patch']
```
Comment (verdurin) [inline easybuild/easyconfigs/f/filevercmp/filevercmp-20191210-GCCcore-10.2.0.eb:16]: @deniskristak please add the missing checksums

File: easybuild/easyconfigs/f/freebayes/freebayes-1.3.5-GCC-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'git_config': {
+        'url': 'https://github.com/freebayes',
+        'repo_name': 'freebayes',
+        'tag': 'v1.3.5',
+        'recursive': True,
+    },
+}]
```
Comment (verdurin) [inline easybuild/easyconfigs/f/freebayes/freebayes-1.3.5-GCC-10.2.0.eb:19]: Checksum should be added here too, if it's consistent.

**Review by verdurin — DISMISSED** [review]

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 6 out of 7 (5 easyconfigs in total)
node379.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/9d5d31f0b5ee40c40d963e5143b232b6 for a full test report.

**Review by lexming — CHANGES_REQUESTED** [review]
I reviewed this PR and it needs some changes. The most important is to only focus on freebayes v1.3.5 and its dependencies, otherwise it will not pass our tests and it's very difficult to find the missing parts. Anything unrelated to freebayes should go in a different PR.

Therefore, I made the following changes in a PR to this one https://github.com/deniskristak/easybuild-easyconfigs/pull/1:
* remove any2fasta and samclip from the PR, they should be moved to your PR about `snippy` https://github.com/easybuilders/easybuild-easyconfigs/pull/13475
* fix the conflicts with filevercmp in current `develop`
* fix build of freebayes:
    * add missing dependencies on `vcflib`, `SeqLib`, `fastahack`, `intervaltree`, `multichoose`, `smithwaterman`
    * patch `meson.build` and set the build environment to properly detect our external dependencies
    * switch the build to a release build
* add easyconfig for SeqLib and its dependencies
* move the update of VCFtools to depend on HTSlib v1.12 from #13477  to this PR

@deniskristak please check the changes in https://github.com/deniskristak/easybuild-easyconfigs/pull/1 and if you agree merge it to update this PR.

**Comment by deniskristak** [comment]
@lexming so rn, I should only add `any2fasta` and `samclip` to https://github.com/easybuilders/easybuild-easyconfigs/pull/13475 , right?

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/a5d653d67de10e1b73192faea02d6dd5 for a full test report.

**Comment by deniskristak** [comment]
@lexming anything left for me to do here?
think you've made most of changes you noted

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Sorry for the delay, this PR is good. I'll check the others that are related to this one.

Going in, thanks @deniskristak !

**Pull request merged**